src/v2/view-builder/views/shared/ChallengeOktaVerifyCustomAppDataView.js

```diff
@@ -1,17 +1,16 @@
+// Common view for OV push and custom push.
 import { BaseFooter } from '../../internals';
 import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
-import 
-ChallengeAuthenticatorDataOktaVerifyPushOnlyFormView
-  from './ChallengeAuthenticatorDataOktaVerifyPushOnlyFormView';
-import ChallengeAuthenticatorDataOktaVerifyFormView from './ChallengeAuthenticatorDataOktaVerifyFormView';
+import ChallengeOktaVerifyCustomPushPushOnlyFormView from './ChallengeOktaVerifyCustomPushPushOnlyFormView';
```

Comment:
name should be `ChallengeOktaVerifyCustomAppPushOnlyFormView ` (since the name of the authenticator is "Custom App")